src/test/java/io/reactivex/TestHelper.java

```diff
@@ -2050,6 +2050,110 @@ protected void subscribeActual(CompletableObserver observer) {
         }
     }
 
+    /**
+     * Check if the given transformed reactive type reports multiple onSubscribe calls to
+     * RxJavaPlugins.
+     * @param transform the transform to drive an operator
+     */
+    public static void checkDoubleOnSubscribeCompletableToFlowable(Function<Completable, ? extends Publisher<?>> transform) {
```

Comment:
Additional double `onSubscribe` call checks with operators going from `Completable` to `Flowable`|`Observable`.